lib/obs_github_deployments/cli/commands/check_lock.rb

```diff
@@ -19,6 +19,8 @@ def call(**options)
 
           response = client(repository: options[:repository], token: options[:token]).locked?
           puts status_response(status: response ? "locked" : "unlocked")
+
+          exit 1 if locked
```

Comment:
`locked` is undefined. I suggest renaming `response` to `locked`. The line 21 will also need to be changed.